### PR TITLE
xbps-install: show overall download progress

### PR DESCRIPTION
**Pull request by sineemore**

### CODE REVIEW

**Comment by sineemore** [comment]
Was proposed in #187

**Comment by sineemore** [comment]
Implementation seems a bit quirky in a way it stores downloaded size.

File: bin/xbps-install/fetch_cb.c [hunk 1]
```diff
@@ -145,9 +168,9 @@ stat_display(const struct xbps_fetch_cb_data *xfpd, void *cbdata)
 		(void)xbps_humanize_number(totsize, (int64_t)xfpd->file_size);
 	}
 	if (v_tty)
-		fprintf(stderr, "%s: [%s %d%%] %s ETA: %s\033[K\r",
-		    xfpd->file_name, totsize, percentage,
-		    stat_bps(xfpd, xfer), stat_eta(xfpd, xfer));
+		fprintf(stderr, "%s %s: [%s %d%%] %s ETA: %s\033[K\r",
```
Comment (sineemore) [inline bin/xbps-install/fetch_cb.c]: Progress is embedded within download line. I've tried adding it as a separate line, but it messes up by package verify message and I don't see a good (a simple) way to address this.

**Comment by xtraeme** [comment]
I just tested this, it works as expected with my last two changes to master.

File: bin/xbps-install/fetch_cb.c (hunk 1 above)
Comment (xtraeme) [reply bin/xbps-install/fetch_cb.c]: I think it's good enough

**Comment by xtraeme** [comment]
@Duncaen ?

**Pull request merged**